pages/guidelines-new.js

```diff
@@ -0,0 +1,383 @@
+import React, { useState } from "react";
+import * as System from "~/components/system";
+import * as Constants from "~/common/constants";
+
+import { css } from "@emotion/react";
+
+import WebsitePrototypeWrapper from "~/components/core/WebsitePrototypeWrapper";
+import WebsitePrototypeHeader from "~/components/core/NewWebsitePrototypeHeader";
+import WebsitePrototypeFooter from "~/components/core/NewWebsitePrototypeFooter";
+
+const STYLES_ROOT = css`
+  padding: 88px 88px 128px 88px;
+  width: 100%;
+  margin: 0 auto;
+
+  @media (max-width: ${Constants.sizes.mobile}px) {
+    display: block;
+    padding: 24px 24px 88px 24px;
+  }
+`;
+
+const STYLES_H1 = css`
+  width: 30%;
+  font-size: ${Constants.typescale.lvl5};
+  font-family: ${Constants.font.medium};
+  font-weight: 400;
+  line-height: 1.3;
+  padding: 0px 0px 16px 0px;
+  letter-spacing: -0.021rem;
+  width: 100%;
+  color: ${Constants.system.slate};
+
+  @media (max-width: ${Constants.sizes.tablet}px) {
+    font-size: ${Constants.typescale.lvl4};
+  }
+
+  @media (max-width: ${Constants.sizes.mobile}px) {
+    font-size: ${Constants.typescale.lvl3};
+  }
+`;
+
+const STYLES_H2 = css`
+  font-size: ${Constants.typescale.lvl3};
+  font-family: ${Constants.font.medium};
+  font-weight: 400;
+  line-height: 1.3;
+  letter-spacing: -0.019rem;
+  margin-top: 24px;
+  width: 100%;
+  color: ${Constants.system.slate};
+
+  @media (max-width: ${Constants.sizes.tablet}px) {
+    font-size: ${Constants.typescale.lvl2};
+  }
+`;
+
+const STYLES_CONTENT_BLOCK1 = css`
+  width: 30%;
+  position: -webkit-sticky;
+  position: sticky;
+  top: 80px;
+
+  @media (max-width: ${Constants.sizes.mobile}px) {
+    position: relative;
+    top: 0;
+    width: 100%;
+  }
+`;
+
+const STYLES_CONTENT_BLOCK2 = css`
+  margin: -264px 0 0 auto;
+  width: 66%;
+
+  @media (max-width: ${Constants.sizes.mobile}px) {
+    margin: 48px 0 0 0;
+    width: 100%;
+  }
+`;
+
+const STYLES_LIST = css`
+  list-style-type: none;
+  font-family: ${Constants.font.text};
+  font-weight: 400;
+  font-size: ${Constants.typescale.lvl1};
+  letter-spacing: -0.011rem;
+  line-height: 1.5;
+  margin: 4px 0 0 16px;
+  opacity: 0.7;
+  padding: 0;
+`;
+
+const STYLES_LINK = css`
+  text-decoration: none;
+  transition: 200ms ease none;
+  color: ${Constants.system.slate};
+
+  :hover {
+    color: ${Constants.system.newBlue};
+  }
+  :active {
+    color: ${Constants.system.newBlue};
+  }
+  :visited {
+    color: ${Constants.system.slate};
+  }
+`;
+
+const STYLES_COPY_EMAIL = css`
+  display: inline;
+`;
+
+export const getServerSideProps = async (context) => {
```

Comment:
Why do you need `getServerSideProps`? is anything coming from the server?

Reply:
actually I'm not sure,,,,,I saw other pages have this so just put it there.

Reply:
This is only necessary if you need to pass props from the server to the client. I don't think you need this for marketing pages so you can delete it.